src/index.css

```diff
@@ -14,8 +14,10 @@ body {
 
 .nav-container {
```

Comment:
- [ ] @MattGomb great work on your design, the color and alignment should match, however, some elements don't match the design given by zeplin, below is a comparison with your design, I have highlighted the sections for easier identification:

Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/202776316-84eb73bc-076d-48f7-b656-40a664fab117.png) |  ![image](https://user-images.githubusercontent.com/61408860/202776702-032eeced-3c54-454a-bc4b-29b1a5ec094e.png)